dao/db.js

```diff
@@ -1,6 +1,3 @@
-const mysql = require('mysql');
-const config = require('../config');
+const db = (process.env.NODE_ENV === 'test') ? require('../config/query.config.test') : require('../config/query.config.prod');
 
-const connection = mysql.createConnection(config);
-
-module.exports = connection;
\ No newline at end of file
+module.exports = db;
```

Comment:
This, too, can be simplified; the following both reduces the horizontal sprawl of the ternary line and avoids an unnecessary variable:

```javascript
if (process.env.NODE_ENV === 'test') {
        module.exports = require('../config/query.config.test');
} else {
        module.exports = require('../config/query.config.prod');
}
```

(Though test all of these suggestions before committing them, of course.)

Reply:
Yeah fair